Streetcode/Streetcode.WebApi/Utils/SoftDeletingUtils.cs

```diff
@@ -20,12 +22,14 @@ public async Task DeleteStreetcodeWithStageDeleted(int daysLife)
             include: s => s.Include(x => x.Observers)
                            .Include(x => x.Targets));
 
-        if (streetcodes is null || streetcodes.Count() == 0)
+        var streetcodeContents = streetcodes.ToArray();
```

Comment:
I cannot. We using `ToArray()` statement for immediate call deferred methods in line `_repositoryWrapper.StreetcodeRepository.GetAllAsync`. After this we just reusing gained result in 2 places, otherwise we needed to execute logic in line `_repositoryWrapper.StreetcodeRepository.GetAllAsync` 2 times instead of 1. By this way we improving code productivity.